### ISSUE TITLE
[BUG]: Polling speed is shown as km/h

### ISSUE THREAD

**Issue by los-cuentos** [CoMPaTech/stromer#152]
### Describe the bug.

The polling speed is shown as km/h instead of minutes or seconds

### What version of Home Assistant Core has the issue?

2024.10.2

### What type of installation are you running?

Home Assistant OS

### How did you install the stromer integration

HACS

**Comment by CoMPaTech** [comment]
Hi @los-cuentos 
Not sure what you mean, the `scan_interval` defaults to 10 minutes (i.e. the time between retrieving data from the API) where the suggestion is to 'trigger' it more frequently by instructing HA to refresh any of the available sensors using an automation. As far as I'm aware there is no 'polling speed' (and/or not sure what you are referring to).

**Comment by los-cuentos** [comment]
I’m sorry for the misunderstanding. I do have a sensor named: sensor.stromer_speed and it keeps fluctuating between 0km/h and 2,5km/h even when the bike is stalled. Though it was the polling speed as u named a similar sensor in the discription to adjust the polling frequency

**Comment by CoMPaTech** [comment]
I'll have a look if/why that sensor is there that way, but you should have a normal speed sensor, right?

**Comment by los-cuentos** [comment]
Yes I do have another speed sensor which is the speed of the bike.
